client/ayon_deadline/plugins/publish/global/validate_deadline_jobinfo.py

```diff
@@ -0,0 +1,38 @@
+import pyblish.api
+
+from ayon_core.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin
+)
+from ayon_deadline.lib import FARM_FAMILIES, get_deadline_pools
+
+
+class ValidateDeadlineJobInfo(
+    OptionalPyblishPluginMixin,
+    pyblish.api.InstancePlugin
+):
+    """Validate collected values for JobInfo section in Deadline submission
+
+    """
+
+    label = "Validate Deadline JobInfo"
+    order = pyblish.api.ValidatorOrder
+    families = FARM_FAMILIES
+    optional = True
+    targets = ["local"]
+
+    # cache
+    pools_by_url = {}
+
+    def process(self, instance):
+        if not self.is_active(instance.data):
+            return
+
+        if not instance.data.get("farm"):
+            self.log.debug("Skipping local instance.")
+            return
+
+        priority = instance.data["deadline"]["job_info"].Priority
+        if priority < 0 or priority > 100:
+            raise PublishValidationError(
+                f"Priority:'{priority}' must be between 0-100")
```

Comment:
It is actually hard limit for DL, it will return nasty broken json response from DL webservice.
But could do validation on attribute definitions, I put this here as a skeleton for potentially other validations. (Couldn't figure any other though, now with dropdowns.)

Reply:
You can pass `minimum=0,maximum=100` to priority attribute def.

Reply:
Reminder ☝️